<launch>

  <node pkg="shared_control" type="fake_robot.py" name="fake_robot" output="screen">
    <param name="pos_x" value="0.8"/>
    <param name="pos_y" value="-1.2"/>
    <param name="pos_th" value="87"/>
    <param name="velocity_lin" value="0.26"/>
    <param name="velocity_ang" value="1.82"/>
    <param name="margin_lin" value="0.1"/>
    <param name="margin_ang" value="0.1"/>
    <param name="sim_cycle" value="0.1"/>
  </node>

  <include file="$(find shared_control)/launch/shared_control.launch"/>

  <node pkg="shared_control" type="fake_bci.py" name="fake_bci" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find shared_control)/rviz/simple_test.rviz"/>

</launch>
